webhook/Makefile

```diff
@@ -0,0 +1,18 @@
+NAME = javaagent-webhook
+IMAGE_PREFIX = hypertrace
+
+export GO111MODULE=on
```

Comment:
nit: not needed as we should use go 15 from start

Reply:
fixed